cpp/ps0001/ps0001solution.cpp

```diff
@@ -1,157 +1,25 @@
-﻿
 #include <iostream>
 #include "hadhod.cpp"
 #include "easylearn.cpp"
 using namespace std;
 
-
-
-
-
-
-
-
-
-
 int main()
 {
+    // prompting user for 3 numbers to add 1/2
     cout << "Enter the first number: \n";
-    float num1;
+    unsigned short num1;
     cin >> num1;
 
     cout << "Enter the second number: \n";
-    float num2;
+    unsigned short num2;
     cin >> num2;
 
     cout << "Enter the third number: \n";
-    float num3;
+    unsigned short num3;

```

Comment:
- [ ] declared 3 variables to store user inputs with data type `unsigned short int`
- [x] used `std::cin` in a right way to prompt user for 3 numbers to add